src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,34 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
-
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { AccordionProps, Accordion } from "./Accordion";
+
+describe("Accordion", () => {
+  
+
+  const renderComponent = (addtionalProps?: Partial<AccordionProps>) => {
+    const defaultProps : AccordionProps = {
+    title: "Click me",
+      children:  "testing",
+    };
+
+    render(<Accordion {...defaultProps} {...addtionalProps} />)
+};
+    it("renders without error", () => {
+        renderComponent();
+        expect(screen).not.toBeNull(); 
+      });
+
+      it('renders the button with the provided text', () => {
```

Comment:
This test does more than the name would suggest, the name says you were testing the accordion title displays correctly, but you're also testing the accordion content shows when you click the accordion button, they should be separate tests.